app/frontend/src/features/auth/signup/CompleteSignupForm.tsx

```diff
@@ -127,71 +130,76 @@ export default function CompleteSignupForm() {
       {loading ? (
         <CircularProgress />
       ) : (
-        <form className={authClasses.form} onSubmit={completeSignup}>
-          <InputLabel className={authClasses.formLabel} htmlFor="username">
-            Username
-          </InputLabel>
-          <TextField
-            className={authClasses.formField}
-            variant="standard"
-            id="username"
-            name="username"
-            fullWidth
-            inputRef={register({
-              pattern: {
-                //copied from backend, added ^ at the start
-                message:
-                  "Username can only have lowercase letters, numbers or _, starting with a letter.",
-                value: usernameValidationPattern,
-              },
-              required: "Enter your username",
-              validate: async (username) => {
-                const valid = await service.auth.validateUsername(username);
-                return valid || "This username is taken.";
-              },
-            })}
-            helperText={errors?.username?.message}
-          />
-          <InputLabel className={authClasses.formLabel} htmlFor="full-name">
-            Full name
-          </InputLabel>
-          <TextField
-            className={authClasses.formField}
-            id="full-name"
-            variant="standard"
-            name="name"
-            fullWidth
-            inputRef={register({
-              pattern: {
-                message: "Name can't be just white space.",
-                value: nameValidationPattern,
-              },
-              required: "Enter your name",
-            })}
-            helperText={errors?.name?.message}
-          />
-          <InputLabel className={authClasses.formLabel} htmlFor="birthdate">
-            Birthday
-          </InputLabel>
-          <Datepicker
-            className={authClasses.formField}
-            control={control}
-            error={!!errors.birthdate}
-            helperText={errors?.birthdate?.message}
-            id="birthdate"
-            inputRef={register({
-              required: "Enter your birthdate",
-              validate: (stringDate) =>
-                validatePastDate(stringDate) ||
-                "Must be a valid date in the past.",
-            })}
-            label={BIRTHDATE_LABEL}
-            minDate={new Date(1899, 12, 1)}
-            name="birthdate"
-          />
-          <InputLabel className={authClasses.formLabel} htmlFor="location">
-            {LOCATION_LABEL}
-          </InputLabel>
+        <>
+          <form
+            className={`${authClasses.form} ${classes.firstForm}`}
+            onSubmit={completeSignup}
+          >
+            <InputLabel className={authClasses.formLabel} htmlFor="username">
+              Username
```

Comment:
There's a few of these hardcoded strings here and the validation messages etc. Can you pull those out to the constants file while you're at these file?

Reply:
Sure, I'm on it!